### ISSUE TITLE
automatic splitting still does not look "the same as before"

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#6743]
From Jenkins tests:
after adding performance report to FJR. predag now works and reports
```
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG Pre-DAG started with output redirected to /data/srv/glidecondor/condor_local/spool/6680/0/cluster8836680.proc0.subproc0/prejob_logs/predag.0.txt
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG found 5 completed jobs
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG jobs remaining to process: 0-1, 0-2, 0-3, 0-4, 0-5
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG jobs remaining to process: 0-1, 0-2, 0-3, 0-4, 0-5
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG average throughput for 5 jobs: 6.804554 evt/s
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG average eventsize for 5 jobs: 57970.8 bytes
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG reduced the target event count from 551168 to 86250.3191262 to obey output size
Sat, 28 Aug 2021 10:19:53 CEST(+0000):DEBUG:JobFactory About to load files by DAO
Sat, 28 Aug 2021 10:19:53 CEST(+0000):DEBUG:JobFactory About to commit 1 jobGroups
Sat, 28 Aug 2021 10:19:53 CEST(+0000):DEBUG:JobFactory About to commit 12 jobs
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG Splitting results:
Sat, 28 Aug 2021 10:19:53 CEST(+0000):INFO:PreDAG Created jobgroup with length 12
```
but previous code was creating only one main job !

to be followed up. But not urgent as long as things work.

**Comment by belforte** [comment]
It also appears that tail jobs are submitted even if main jobs completed OK.

Maybe leave automatic splitting for a second approximation after everything else works

**Comment by belforte** [comment]
subdag_1 log has
```
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Pre-DAG started with output redirected to /data/srv/glidecondor/condor_local/spool/6680/0/cluster8836680.proc0.subproc0/prejob_logs/predag.1.txt
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG found 12 completed jobs
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG jobs remaining to process: 1, 10, 11, 12, 2, 3, 4, 5, 6, 7, 8, 9
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG jobs remaining to process: 1, 10, 11, 12, 2, 3, 4, 5, 6, 7, 8, 9
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG average throughput for 12 jobs: 4.66895083333 evt/s
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG average eventsize for 12 jobs: 57966.9166667 bytes
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG reduced the target event count from 100849 to 86256.0972279 to obey output size
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 11
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 10
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 12
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 1
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 3
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 2
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 5
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 4
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 7
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 6
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 9
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Adding missing lumis from job 8
Sat, 28 Aug 2021 11:05:33 CEST(+0000):DEBUG:JobFactory About to load files by DAO
Sat, 28 Aug 2021 11:05:33 CEST(+0000):DEBUG:JobFactory About to commit 1 jobGroups
Sat, 28 Aug 2021 11:05:33 CEST(+0000):DEBUG:JobFactory About to commit 12 jobs
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Splitting results:
Sat, 28 Aug 2021 11:05:33 CEST(+0000):INFO:PreDAG Created jobgroup with length 12
```
so it looks like FJR still does not have the needed information.

**Comment by belforte** [comment]
could maybe be due maxEvent having slipped in ? The 12 main jobs processed 10 events each !

will try automatic splitting again after #6740 is fixed.

**Comment by belforte** [comment]
hopefully solved via the fix to #6740 in https://github.com/dmwm/CRABServer/pull/6744
